rebar.config

```diff
@@ -15,6 +15,7 @@
        , {yamerl,      "0.7.0"}
        , {docsh,       "0.7.2"}
        , {worker_pool, "4.0.1"}
+       , {elvis,       "0.4.2", {pkg, elvis_core}}
```

Comment:
Why not just declare the dependency as `elvis_core`? Also, this is already part of the `test` profile. Now that we include it everywhere we should probably remove it from there.

Reply:
I copied this from the documentation, will simplify it.